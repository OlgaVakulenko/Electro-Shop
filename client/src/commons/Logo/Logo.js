import React from 'react';
import Box from '@material-ui/core/Box';

import { useStyles } from './styles';

export function Logo() {
  const classes = useStyles();
  return (
    <Box className={classes.logo}>
      <img src="/img/logo.svg" alt="Logo" />
    </Box>
  );
}
