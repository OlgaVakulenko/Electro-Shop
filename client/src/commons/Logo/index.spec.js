import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from './Logo';

describe('Logo', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Logo />,
    );
    expect(output).toMatchSnapshot();
  });
});
