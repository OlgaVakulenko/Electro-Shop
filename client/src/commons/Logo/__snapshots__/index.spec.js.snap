// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly 1`] = `
<Link
  to="/"
>
  <Styled(MuiBox)
    className="makeStyles-logo-1"
  >
    <img
      alt="Logo"
      src="/img/logo.svg"
    />
  </Styled(MuiBox)>
</Link>
`;
